htmresearch/frameworks/classification/network_factory.py

```diff
@@ -31,6 +31,13 @@
 from nupic.engine import Network
 from nupic.engine import pyRegions
 
+try:
+  import capnp
+except ImportError:
+  capnp = None
+if capnp:
+  from nupic.proto import NetworkProto_capnp
```

Comment:
That's right, good catch. I had initially taken this out of [this example](https://github.com/numenta/nupic/blob/master/tests/integration/nupic/engine/network_creation_common.py#L36-L41) to try out capnp serialization but if `capnp` is unimportable,  `loadNetwork()` and `saveNetwork()` would throw errors. I am going to remove these imports and methods - they are unused because model serialization does not work at the moment.